src/Lookup/LabelLookup.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Wikibase\DataModel\Services\Lookup;
+
+use Wikibase\DataModel\Entity\EntityId;
+use Wikibase\DataModel\Term\Term;
+
+/**
+ * @since 3.9
+ *
+ * @license GPL-2.0+
+ * @author Thiemo Mättig
+ */
+interface LabelLookup {
+
+	/**
+	 * @since 2.0
```

Comment:
Actually I'm not convinced it makes sense to have a tag at all on method level in a codebase where you have sane interfaces. And given there is just one method here...